src/SFA.DAS.ApprenticeCommitments.Web/Startup/ApplicationStartup.cs

```diff
@@ -27,6 +27,7 @@ public void ConfigureServices(IServiceCollection services)
                 .AddAuthentication(appConfig.Authentication, Environment)
                 .AddOuterApi(appConfig.ApprenticeCommitmentsApi)
                 .AddHashingService(appConfig.Hashing)
+                .AddSharedUi(appConfig)
```

Comment:
Could this be appConfig.Zendesk?

Reply:
It *could* be.  I was thinking that we will probably have further config for the Shared UI later - e.g the link for the feedback form (in the beta banner)